<launch>
    <node name="relative_position" pkg="RoNAOldo" type="RoNAOldo_relative_position" output="screen">
        <remap from="goalPositionTop" to="/goalPosition" />
        <remap from="ballTop" to="/ball" />
    </node>

    <node name="ball_vision_top" pkg="RoNAOldo" type="RoNAOldo_ballVision" output="screen">
        <remap from="image" to="/nao/nao_robot/camera/top/camera/image_raw" />
    </node>

    <node name="marker_vision_top" pkg="RoNAOldo" type="RoNAOldo_vision_marker" output="screen">
        <remap from="image" to="/nao/nao_robot/camera/top/camera/image_raw" />
    </node>
</launch>
